### DISCUSSION TITLE
WhiteBoard User

### DISCUSSION THREAD

**Discussion by Merlinux24** [community/community#139947] [New to GitHub]
Hello, 

I am a Newbie here just because I am using Ms-WhiteBoard with which I cannot log in with my usual Microsoft account from an android device although that works upon other ones like on smartphone with android or on iphone with IOS for instance.

So I signed in by github in order to be able to log in with it directly there but the trouble remains as  following.

![image](https://github.com/user-attachments/assets/6cc9eaa7-42d4-449b-8543-a0e4cbad9b01)

I found no solution in internet to solve this trouble with that android app. However it works only upon a browser like Chrome. But it is impossible to paste there a picture. That 's  why I would like to use the app in the one android device.

So I am confused now and in fact I do not know what I have to do or can do by GitHub... although I am myself, a former developer with classical tools like Basic, C, Pascal, Scripts. Java and so on, even still today with Excel or Words  ... ;-)

Regards

**Comment by AidenJamesYt** [comment]
Could you provide a translation of the screenshot?

**Reply by Merlinux24** [reply]
**Screenshot translation**
************************
[Login Error]
Something went wrong at login - unknown error - try again,
Correlation Id: 35e30382-e538-4470-93f9- (_this changes at trying again_)
Try to launch the app again.
[Repeating]

**Reply by Merlinux24** [reply]
In the meantime I got from the Microsoft community with the suggestion to try a proxy of another region. However I do not know why this could be an issue, when I only can connect with the chrome browser and not with the android app. Nor  which free proxy I should use, for it does not work from some countries I choose from the list e.g. from spy.one.

**Reply by AidenJamesYt** [reply]
could be a blocked proxy
